src/main/java/roomescape/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,15 @@
+package roomescape.exception;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(value = IllegalArgumentException.class)
```

Comment:
아래와 같이 value 는 생략해주셔도 좋아요!
(물론 개인 취향입니다 ㅎㅎ)

```suggestion
@ExceptionHandler(IllegalArgumentException.class)
```

Reply:
훨씬 깔끔하네요! 감사합니다. 반영했습니다. ☺️